packages/apps-config/src/ui/logos/index.ts

```diff
@@ -380,6 +380,7 @@ export const nodeLogos = Object.entries({
   'DOTMog Node': nodeDotMog,
   'Eave Node': nodeEave,
   'Edgeware Node': nodeEdgeware,
+  'Efinity': nodeEfinity,
```

Comment:
```suggestion
  Efinity: nodeEfinity,
```

Fixes https://github.com/polkadot-js/apps/runs/5232306403?check_suite_focus=true#step:3:700